### DISCUSSION TITLE
Does this boilerplate support bundling native libraries (like sqlite) ?

### DISCUSSION THREAD

**Discussion by t13m** [cawa-93/vite-electron-builder#598] [Q&A]
Hello, I was wondering if this boilerplate support bundling native libraries (like sqlite). If so, any kind of example or instructions would be very helpful!

Thanks

**Comment by cawa-93** [comment]
I didn't check it. But as far as I know, the electron builder will package native dependencies. The template itself does nothing on top of that.

**Comment by antholord** [comment]
I've tried using a library that has native code, like node-window-manager, and it worked without issue. As Cawa said, Electron-builder takes care of building and packaging native deps. This is great because in most of the other Electron boilerplates that use webpack, it's not that simple.
